zoom_operation_rules/zoom_operation_user_granted_admin.py

```diff
@@ -0,0 +1,18 @@
+from panther_zoom_helpers import get_zoom_user_context as get_context
+
+
+def rule(event):
+    if event.get("category_type") != "User" and event.get("Action") == "Update":
+        return False
+
+    context = get_context(event)
+
+    if "Member to Admin" in context["Change"]:
+        return True
+    return False
```

Comment:
```suggestion
    return "Member to Admin" in context["Change"]:
```

Reply:
Much cleaner, thank you. I always forget to go back and clean up stuff like this